conda_forge_tick/migrators/cstdlib.py

```diff
@@ -250,4 +250,4 @@ def migrate(self, recipe_dir, attrs, **kwargs):
             )
 
             with open(fname, "w") as fp:
-                fp.write("".join(cbc_lines))
+                fp.write("".join(cbc_lines) + "\n")
```

Comment:
I think this adds a newline every time the file is rewritten, even if there is one already, so it would grow on each migration.

I also think this code also didn't have a problem stripping newlines since `cbc_lines` would have trailing newlines preserved, so I don't think it was related to #2696